app/Blogs/page.js

```diff
@@ -0,0 +1,34 @@
+"use client";
+
+import Link from "next/link";
+import { useRouter } from "next/navigation";
+
+export default function Blog() {
+  const blogPosts = [
+    { title: "My First Post", slug: "my-first-post" },
+    { title: "Learning React", slug: "learning-react" },
```

Comment:
This static data can be outside of the component. Also have a look at [generateStaticParams](https://nextjs.org/docs/app/api-reference/functions/generate-static-params)

Reply:
Noted